src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderTransactionHistory.java

```diff
@@ -78,6 +90,32 @@
     @Default
     private List<OffenderTransactionDetails> relatedTransactionDetails = new ArrayList<>();
 
+    @Builder.Default
+    @Transient
+    private BigDecimal currentBalance = BigDecimal.ZERO;
+
+    @Column(name = "CREATE_DATETIME")
+    @CreatedDate
+    private LocalDateTime createDatetime;
```

Comment:
The createDatetime property in AuditableEntity was set to protected scope so I couldn't access it via the builder. I tried creating a custom getter and setter that would delegate to super.createDateTime but the Lombok builder still wouldn't pick it up. So in the end I moved over the audit props and exposed the one I need.

Reply:
Well, I guess you tried it but could we add a method with a different name, e.g:
    public LocalDateTime getCreatedDateTime() {
        return getCreateDatetime();
    }

Reply:
I did try that but the new getter wouldn't seem to conform to the Lombok builder interface.  Maybe I messed it up.